tutorials/getting_started/using_pretrained_models.md

```diff
@@ -0,0 +1,68 @@
+# Using Pretrained AllenNLP Models
+
+In this tutorial, we show how to use run the pretrained models in AllenNLP to make predictions.
+While this tutorial will specifically use the Named Entity Recognizer, but will apply to any of
+the models [available on our website](http://allennlp.org/models).
```

Comment:
this sentence is awkward

"This tutorial uses the Named Entity Recognition model, but the same procedure applies to any of the model available on our website"